### ISSUE TITLE
Animation import should allow setting boundaries for the usable frames.

### ISSUE THREAD

**Issue by Menotdan** [FlaxEngine/FlaxEngine#2366]
Currently, when you import an animation, Flax will use all the frames, but it would be helpful to add start and end frames in the import settings. Flax currently lets you change what frame the animation ends, but not when it starts. The start position option of animation nodes in the graph could technically work but it's much more confusing.

**Comment by mafiesto4** [comment]
When displaying import options dialog we don't read the source file so we don't know it's contents. To solve this issue we would need to add support for quick peeking the contents like animations count and frame ranges.

**Comment by Menotdan** [comment]
> When displaying import options dialog we don't read the source file so we don't know it's contents. To solve this issue we would need to add support for quick peeking the contents like animations count and frame ranges.

Yes. I think this is also valuable for many other features, for example, excluding meshes by name from the import if you want, or you can mark collision meshes as such and have flax import them.
